rollup/internal/controller/watcher/batch_proposer.go

```diff
@@ -155,6 +155,28 @@ func (p *BatchProposer) TryProposeBatch() {
 }
 
 func (p *BatchProposer) updateDBBatchInfo(batch *encoding.Batch, codecVersion encoding.CodecVersion, metrics utils.BatchMetrics) error {
+	for {
+		compatible, err := utils.CheckBatchCompressedDataCompatibility(batch, codecVersion)
+		if err != nil {
+			log.Error("Failed to check batch compressed data compatibility", "err", err)
```

Comment:
```suggestion
			log.Error("Failed to check batch compressed data compatibility", "codecVersion", codecVersion,  "err", err)
```

Reply:
added this info in [0944e49](https://github.com/scroll-tech/scroll/pull/1431/commits/0944e498da4a1a2f6b785bf1849202819a0335f0).